docs/integrations/app-connections/aws.mdx

```diff
@@ -259,7 +261,9 @@ Infisical supports two methods for connecting to AWS.
                                     "ssm:GetParametersByPath",
                                     "ssm:DescribeParameters",
                                     "ssm:DeleteParameters",
+                                    "ssm:ListTagsForResource", // if you need to add tags to secrets
                                     "ssm:AddTagsToResource", // if you need to add tags to secrets
+                                    "ssm:RemoveTagsFromResource", // if you need to add tags to secrets
                                     "kms:ListKeys", // if you need to specify the KMS key
```

Comment:
thoughts on displaying a message when KMS key list is empty? something to inform them that the permissions they configured for the attached IAM is missing the ListKeys or ListAliases

<img width="632" alt="image" src="https://github.com/user-attachments/assets/b2365652-f2b5-45c6-abb1-1e8ccf2f0ab8" />